Let's try to add a label, and make new commits in a pull request

illgidgud opened the pull request.

illgidgud committed:
Create MyFile

illgidgud committed:
Delete MyFile

illgidgud committed:
Changed entry1.

NgocHieu0912 reviewed (commented):
Seems like i'll need to update your access to triage or above.

illgidgud committed:
Pull request update

I made an extra commit to see whether the existing pull request will be updated or not.

NgocHieu0912 commented:
It does update new changes to the pull request !

NgocHieu0912 committed:
Added a file on git. Let's see if it has the same effect as adding files on the web.

NgocHieu0912 committed:
Deleted the files. They have done their jobs.
